### PR TITLE
[vcenter] instantiate as persistent, reduce overhead, starting bootOrder

### PR DESCRIPTION
**Pull request by Semedi**
**NO SQUASH**

### CODE REVIEW

**Comment by Semedi** [comment]
only master

File: src/rm/RequestManagerClone.cc [hunk 1]
```diff
@@ -184,7 +184,7 @@ Request::ErrorCode VMTemplateClone::clone(int source_id, const string &name,
             oss << name << "-disk-" << ndisk;
 
             ec = img_clone.request_execute(img_id, oss.str(), -1,
-                    (*disk)->is_managed(), new_img_id, img_att);
+                    true, new_img_id, img_att);
```
Comment (tinova) [inline src/rm/RequestManagerClone.cc]: maybe we don't need this parameter anymore @rsmontero ?

**Comment by tinova** [comment]
lgtm but ideally would want to have @rsmontero review the core change

File: src/rm/RequestManagerClone.cc (hunk 1 above)
Comment (Semedi) [reply src/rm/RequestManagerClone.cc]: i think there is another call using the parameter, thats why im not removing it.

**Comment by Semedi** [comment]
need to talk with @rsmontero before the merge.

**Pull request merged**